packages/pagination/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@availity/pagination",
-  "version": "2.12.0",
+  "version": "2.13.0",
```

Comment:
Did you bump this manually? You can move this back down to `2.12.0` and the github actions will handle bumping the version and publishing when we merge the pr

Reply:
Yes, I did - lowered it back now. I will remember that going forward!